tutorials/haskell/golden-testing-in-haskell-with-hspec-gold/tutorial.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Golden testing in Haskell with Hspec-Golden
+published: 2016-08-10
```

Comment:
@MedAdrian Don't forget to update this date.

Reply:
I don't think i know the date for the release, it's okay if i put '2019-08-10'?

Reply:
@MedAdrian You're right, we don't know that yet. So, we can wait a little bit on that.